scripts/cve_message.py

```
@@ -0,0 +1,496 @@
+    msg: str
+        Message containing vulnerability summary.
+    """
+
+    url = os.getenv("INFRA_WEBHOOK_URL")
+    try:
+        response = requests.post(
+            url,
+            json={"text": msg}
+        )
+    except Exception:
+        print(f"Response: {response.status_code}")
```

Pylint (3.3.7)

[error] Using variable 'response' before assignment

(E0601)